src/main/java/org/apache/commons/imaging/formats/tiff/TiffField.java

```diff
@@ -234,6 +234,14 @@ public int[] getIntArrayValue() throws ImagingException {
             final int[] numbers = (int[]) o;
             return Arrays.copyOf(numbers, numbers.length);
         }
+        if (o instanceof long[]){
```

Comment:
The formatting seems off in the block, missing spaces.

Reply:
done